cli/src/commands/generate/generate.ts

```diff
@@ -46,6 +46,30 @@ function getPropertyValue(keyName: string, detail: any) : any {
     }
 }
 
+function instantiateNodeInterfaces(detail: any): any[] {
+    const interfaces = [];
+    if (!('prefixItems' in detail)) {
```

Comment:
we should definitely revisit this once we've figured out what we're doing with our typed representation of CALM. `any` is a virus which will bleed into every corner the codebase if we're not careful, then we may as well use JS